ontology/obcs.md

```diff
@@ -22,10 +22,13 @@ tags:
 - statistics
 tracker: https://github.com/obcs/obcs/issues
 usages:
+- description: VEuPathDB Clinical Epidemiology Resources use OBCS
   examples:
   - description: OBCS is used in annotation of study designs
     url: https://clinepidb.org/ce/app/search/dataset/Studies/result
   user: https://clinepidb.org/
+- description: OBCS is used in statistical analysis method annotation and analysis in literature text mining
+  user: https://link.springer.com/article/10.1007/s40484-017-0122-5
```

Comment:
I can't comment on whether these usages are adequate or not

Reply:
Thanks, @cthoyt. Who do you think I can request to review?